linalg/densemat.cpp

```diff
@@ -4010,7 +3789,8 @@ void CholeskyFactors::LSolve(int m, int n, real_t * X) const
    int info = 0;
 
 #ifdef MFEM_USE_SINGLE
-   strtrs_(&uplo, &trans, &diag, &m, &n, data, &m, X, &m, &info);
```

Comment:
`#ifdef MFEM_USE_SINGLE` should be removed (looks you only missed one!).

Reply:
Fixed in 262fa6173d8307be5e25bc2f945ef43dfd368174